### PR TITLE
feat(i18n): add i18n configuration (French translation + language selector in user settings)

### PR DESCRIPTION
**Pull request by dq18**
### What
- Added translations (for now only English and French are available, Spanish support is added but nothing is translated)
- Added Language selection in Settings

### Todo
- Warning shows up: [intlify] Not found parent scope. use the global scope.
- Pluralization not working
- Add other languages

### CODE REVIEW

**Comment by teolemon** [comment]
I added a Crowdin config file

**Comment by dq18** [comment]
> I added a Crowdin config file

Hmm je viens juste de voir ça. Il y avait un problème de check et j'ai fait un revert commit. Donc la config file n'est plus dedans.
On squash merge cette PR et on fait ça sur une autre? C'est déjà bien compliqué sur celle là (ma faute).

File: src/views/UserSettings.vue [hunk 1]
```diff
@@ -1,18 +1,43 @@
...
   <v-form @submit.prevent="updateSettings">
     <v-row>
       <v-col cols="12" sm="6">
-        <v-card title="Adding prices" prepend-icon="mdi-tag-plus-outline">
+        <v-card :title="$t('UserSettings.ChangeLanguage')" prepend-icon="mdi-earth">
           <v-divider></v-divider>
           <v-card-text>
-            <h3 class="mb-1">Default currency</h3>
+            <v-autocomplete
+              v-model="userSettingsForm.selectedCountry"
```
Comment (raphodn) [inline src/views/UserSettings.vue:13]: this seems overkill (and not working) for the v1 of the translation mecanism :)

I'd expect a simple language selector (with only the `constants.LANGUAGE_LIST` + stored in the store)

**Review by raphodn — APPROVED** [review]
Awesome ! 

I did some minor changes : 
- https://github.com/openfoodfacts/open-prices-frontend/pull/154/commits/a57da430839b4a7876812df65eeb3a4f65897b2b : add new lines, replace `"` with `'`, remove `;` (we need a pre-commit :sweat_smile: )
- https://github.com/openfoodfacts/open-prices-frontend/pull/154/commits/2a747d69031ade160eadd092327c2f2510be137e : added and edited a few translations

File: src/components/PriceCard.vue [hunk 2]
```diff
@@ -37,7 +37,13 @@
             <p>
               <span>{{ getPriceValueDisplay() }}</span>
               <span v-if="hasProductQuantity"> ({{  getPricePerKilo() }})</span>
-              <span> on <i>{{ getDateFormatted(price.date) }}</i></span>
+              <span>
+                <i18n-t keypath="PriceCard.PriceDate" tag="p">
```
Comment (raphodn) [inline src/components/PriceCard.vue:41]: why the `tag="p"` ? I see it elsewhere, it's making new lines

Comment (raphodn) [reply src/components/PriceCard.vue:41]: fixed here : https://github.com/openfoodfacts/open-prices-frontend/commit/b5d4282cebe574f6d973d076489c51196d92a25d

**Pull request merged**